src/test/java/com/google/api/codegen/testdata/discogapic/java/java_simplecompute.v1.json.baseline

```diff
@@ -28,11 +28,12 @@ import javax.annotation.Generated;
 
 @Generated("by GAPIC")
 @BetaApi
-public final class AddressesRegionName implements ResourceName {
+public final class AddressName implements ResourceName {
+  private final String address;
   private final String project;
   private final String region;
   private static final PathTemplate PATH_TEMPLATE =
-        PathTemplate.createWithoutUrlEncoding("{project}/regions/{region}/addresses");
+        PathTemplate.createWithoutUrlEncoding("project/{project}/regions/{region}/addresses/{address}");
```

Comment:
this should start with `projects` (plural)

Reply:
oooh good catch; done.